libfreerdp/core/rdp.c

```diff
@@ -1617,37 +1617,39 @@ static int rdp_recv_pdu(rdpRdp* rdp, wStream* s)
 		return rc;
 }
 
-int rdp_recv_callback(rdpTransport* transport, wStream* s, void* extra)
+/* TODO: Need to properly return:
```

Comment:
Since we are already touching large parts of this code - can we replace these magic numbers with proper defines so everybody knows immediately what is going? (same in peer.c)

Reply:
@mfleisz good idea, added an enum and a string function for it